ui-lib/src/main/java/co/electriccoin/zcash/ui/screen/home/viewmodel/WalletViewModel.kt

```diff
@@ -241,6 +245,63 @@ sealed class SecretState {
     class Ready(val persistableWallet: PersistableWallet) : SecretState()
 }
 
+/**
+ * Represents all kind of Synchronizer errors
+ */
+sealed class SynchronizerError {
```

Comment:
I created a followup issue for this, so that we can localize these https://github.com/zcash/secant-android-wallet/issues/529